src/components/search/sort.vue

```diff
@@ -18,7 +18,7 @@
         <template #default="{ item, isSelected }">
           <RadioButtonSelectedIcon v-if="isSelected" class="x-icon--l" />
           <RadioButtonUnselectedIcon v-else class="x-icon--l" />
-          <span class="x-font-color--neutral-10" :class="{ 'x-text--bold': isSelected }">
+          <span class="x-text-neutral-90" :class="{ 'x-text--bold': isSelected }">
```

Comment:
Should we remove that `x-text--bold` class?

Reply:
We should change it to `x-font-bold` yes